Libraries/Animated/src/AnimatedImplementation.js

```diff
@@ -312,8 +312,8 @@ class TimingAnimation extends Animation {
 
   stop(): void {
     this.__active = false;
-    clearTimeout(this._timeout);
-    window.cancelAnimationFrame(this._animationFrame);
+    this._timeout && clearTimeout(this._timeout);
+    this._animationFrame && window.cancelAnimationFrame(this._animationFrame);
```

Comment:
Should be fine to leave this -- clearTimeout & cancelAnimationFrame are designed to accept null or inactive timer handles.